docs/import-roster-from-lms.md

```diff
@@ -0,0 +1,94 @@
+## Importing a course roster from your Learning Management System
+
+GitHub Classroom can automatically import your course roster from your institution's learning management
+system to save you time when creating a GitHub Classroom course.
+
+### Prerequisites
+
+If you haven't yet, [connect GitHub Classroom to you learning management system](/help/connect-to-lms) before proceeding.
+
+Importing a course roster from your learning management system will only work if your learning management system's privacy settings allow GitHub Classroom to receive course roster information.
+
+### Creating a new course roster
+
+To import a new course roster from your learning management system:
+
+   <div class="d-flex flex-justify-around">
+     <img src="/images/help/lms/roster-import/import-roster.gif" class="border" style="width: 75%;">
+   </div>
+
+1. [Sign in to GitHub Classroom](https://classroom.github.com/login).
+
+2. Select the classroom you wish to import your roster into.
+
+3. Navigate to _Classroom Settings_ within GitHub Classroom
+
+4. In the _sidebar_, select **Roster Management**. A page named _Add Students_ appears.
+
+5. Click **Import from your Learning Management System**
+
+6. Select how you would like to identify the imported students by:
+
+    - **User ID**: The Student ID as stored in your institution's learning management system
+    - **Names**: Student full names
+    - **Emails**: Student emails
+
+    **Note**: If you do not see the kind of identifier you want, you may have to enable GitHub Classroom to receive student name and email information in your learning management system's privacy settings. See [troubleshooting](#troubleshooting) for more advice.
+    {: class="note"}
+
+You have imported your roster into GitHub Classroom.
+
+  **Note**: If you encounter any errors, check to ensure you have enabled the roster membership service from within your learning management system. See [troubleshooting](#troubleshooting) for more advice.
+  {: class="note"}
+
+
+### Syncing an existing course roster
+
+After you've imported a roster, keep it up to date by syncing:
+
+   <div class="d-flex flex-justify-around">
+     <img src="/images/help/lms/roster-import/sync-roster.gif" class="border" style="width: 75%;">
+   </div>
+
+1. [Sign in to GitHub Classroom](https://classroom.github.com/login).
+
+2. Select the classroom you wish to import your roster into.
+
+3. Navigate to _Classroom Settings_ within GitHub Classroom.
+
+4. In the _sidebar_, select **Roster Management**. You will be directed to your existing course roster.
```

Comment:
A little tense thing to watch out for here: understandably, we think of it in the future tense—the reader will do all these things—but the reader experiences it in the present tense

```suggestion
4. In the sidebar, select **Roster Management**. You are directed to your existing course roster.
```